packages/components/buttons/flat-button/src/flat-button.tsx

```diff
@@ -7,19 +6,18 @@ import {
   ComponentPropsWithRef,
   cloneElement,
 } from 'react';
-import { css, useTheme } from '@emotion/react';
+import { css } from '@emotion/react';
 import omit from 'lodash/omit';
-import { customProperties as vars } from '@commercetools-uikit/design-system';
```

Comment:
I took chance to change how we import `customProperties` object to 
```js
import { customProperties } from '@commercetools-uikit/design-system';
```
If however that is not needed, please let me know. 
Also, please let me know if you wish me to refactor all the remaining components.